### PR TITLE
#787 add padding or margin to main so content dont overlap with header

### PR DESCRIPTION
**Pull request by mathiasg12**
## Describe your changes: 
- added pt-5 to all index.html files (this did not work as it should)
- removed pt-5 from all index.html files
- added min-height to header of 86px
- added padding-top to main of 86px

## Type of changes:
Bootstrap change
SCSS change

## Checklist before requesting a review

* [x ] I created a new branch before starting with this ticket?
* [x ] I commented the issue after finish in the Frontend Team Board?
* [x ] I moved the card to Quality Assurance after finish?

## Did you run into any issues?
- NO

## Screenshots:
main
header

### CODE REVIEW

**Review by ESPR07 — CHANGES_REQUESTED** [review]
This still doesn't fix the issue, and is not a very responsive way of fixing it.
bilde

I would either change the padding to be a responsive unit like VW or VH or change the position attribute on the nav to be sticky instead of fixed to retain the document flow.

**Comment by mathiasg12** [comment]
I will try to find another way around this

**Review by ESPR07 — APPROVED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-  "liveServer.settings.port": 5501
+  "liveServer.settings.port": 5501,
+  "editor.formatOnSave": false,
```
Comment (kyrregjerstad) [inline .vscode/settings.json:3]: any particular reason you disabled `format on save`? It's disabled for everyone now ;)

Comment (mathiasg12) [reply .vscode/settings.json:3]: sorry forgot to undo it, had some weird formating happening every time I saved so "files changed" would be huge when often only a few line of code was added or removed

Comment (kyrregjerstad) [reply .vscode/settings.json:3]: no problem, could you make a new PR with the fix? :)

**Pull request merged**